config/locales/inherited_proofing/en.yml

```diff
@@ -23,15 +23,15 @@ en:
     errors:
       cannot_retrieve:
         failure_info: Please check back later.
-        heading: We could not retrieve your information from My HealtheVet
+        heading: We could not retrieve your information from your Partner Agency
```

Comment:
Since we have the SP name, Why can't we pass in the Partner agency name here for a nicer error?

Reply:
We can, and should. Dealing with the hard-coded url's was beyond the scope of this card, but I added the presenter to at least get all that service provider agnostic. I'll put in a card to make sure we use the presenter in a more efficient manner.